### ISSUE TITLE
[feature request] minlen for xwindow module

### ISSUE THREAD

**Issue by protoroto** [jaagr/polybar#349]
I want to make the width of the xwindow module fixed, so that it will always keep the same width despite what's written in it. The maxlen parameter is helpful for not making the module too wide, but it would be great if also a minlen parameter would be implemented, or maybe you can achieve what I'm trying to do in some other way. Is this something you can think about or it's a silly request? 

By the way, this bar is awesome. For real.

**Comment by jaagr** [comment]
This should already be possible using token specifiers.
[See the wiki for more details](https://github.com/jaagr/polybar/wiki/Formatting#tokens)

**Comment by protoroto** [comment]
You're totally right. No need for anything new.

**Comment by derrgill** [comment]
> This should already be possible using token specifiers.
> [See the wiki for more details](https://github.com/jaagr/polybar/wiki/Formatting#tokens)

I'm not sure what I'm doing wrong, I followed the Wiki section mentioned above, but ...
`label = %percentage:3%`
... does not seem to keep my coreuse module (nor RAM and swap modules) from _shifting_ other modules left when the percentage goes over 9% or 99%

I also tried the following snippets (althought the Wiki describes these as "unreleased"), to no avail as well:
`label-minlen = 3
label-alignment = right`

Am I doing something incorrectly? Also, is there any timeline for release the "label-minlen" feature?

Thanks!
